src/modules/flight_mode_manager/tasks/Auto/FlightTaskAuto.hpp

```diff
@@ -204,6 +204,7 @@ class FlightTaskAuto : public FlightTask
 	WeatherVane _weathervane{this}; /**< weathervane library, used to implement a yaw control law that turns the vehicle nose into the wind */
 
 	matrix::Vector3f _initial_land_position;
+	bool _land_position_modified{false};
```

Comment:
Am I blind or what is `_land_position_modified` for? How can it ever get rue? 👀

Reply:
You're right, it's never set to true and it works without this; I'll remove it.